commands.py

```diff
@@ -17,6 +33,24 @@ class CopilotTextCommand(LspTextCommand, metaclass=ABCMeta):
     def want_event(self) -> bool:
         return False
 
+    def _record_telemetry(
+        self, request: str, payload: Union[CopilotPayloadNotifyAccepted, CopilotPayloadNotifyRejected]
+    ) -> None:
+        session = self.session_by_name(self.session_name)
```

Comment:
I would add
```python
if not get_setting(session, "telemetry", False):
    return
```
to avoid even getting a session if the `telemetry` set to `false`

Reply:
We pull session settings instead of view settings. Perhaps it doesn't matter here though?

Reply:
Hm, indeed, you are right